test/test_transforms_v2_refactored.py

```diff
@@ -308,36 +308,26 @@ def wrapper(input, *args, **kwargs):
     return wrapper
 
 
-def make_input(input_type, *, dtype=None, device="cpu", spatial_size=(17, 11), mask_type="segmentation", **kwargs):
+def make_input(input_type, *, spatial_size=(17, 11), mask_type="segmentation", **kwargs):
```

Comment:
Why move `dtype` and `device` to `kwargs` and not `spatial_size`?
I'm starting to wonder whether we need `make_input` at-all. We could parametrize over `make_xyz` instead of parametrizing over the `input_type` parameter.

Reply:
> Why move `dtype` and `device` to `kwargs` and not `spatial_size`?

Because `make_input` currently has a default value for `spatial_size`, but the lower level functions do not. Will change that.